application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -41,12 +45,24 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
-        // TODO A list might be better than comma separated, which is hard to read
+
+        ArrayList<String> list = new ArrayList<>(tagSystem.getAllIds());
```

Comment:
* `List` not `ArrayList`
* `list` is a bad method name. It should be named according to what it contains, `tags` or `tagIds` for example.

Reply:
That said, why are you even creating a new list? Just work with what `getAllIds()` returns directly.

Reply:
Yes I agree with you,  il find another way of doing it  👍